### DISCUSSION TITLE
Builds missing for the v1.4.5 release

### DISCUSSION THREAD

**Discussion by ckujau** [manugarg/pacparser#202] [General]
Version 1.4.3 has builds available for macOS, Ubuntu Linux and Windows (after @tech189 mentioned [they were missing](https://github.com/manugarg/pacparser/discussions/186#discussioncomment-8169641)). And now for [v1.4.5](https://github.com/manugarg/pacparser/releases/tag/v1.4.5) it's missing again :-)

The [build job](https://github.com/manugarg/pacparser/actions/runs/9727458474) looks fine, maybe someone can push the binaries to the release page?

**Comment by manugarg** [comment]
Done.

I was so focussed on python packages (pypi)  and docker image, I didn't upload the binaries. Also, there was no functionality change in this release. It was purely a release to add more python builds.

**Comment by ckujau** [comment]
Wow, that was quick! Thank you so much!

**Comment by ckujau** [comment]
Is this a [Matryoshka](https://en.wikipedia.org/wiki/Matryoshka_doll) release? :-)
```
$ curl -sLO https://github.com/manugarg/pacparser/releases/download/v1.4.5/pacparser-v1.4.5-ubuntu-x86_64.zip
$ unzip pacparser-v1.4.5-ubuntu-x86_64.zip 
Archive:  pacparser-v1.4.5-ubuntu-x86_64.zip
  inflating: pacparser-v1.4.5-ubuntu.zip  

$ unzip pacparser-v1.4.5-ubuntu.zip
Archive:  pacparser-v1.4.5-ubuntu.zip
   creating: pacparser-v1.4.5-ubuntu/
   creating: pacparser-v1.4.5-ubuntu/lib/
  inflating: pacparser-v1.4.5-ubuntu/lib/libpacparser.so  
  inflating: pacparser-v1.4.5-ubuntu/lib/libpacparser.so.1  
[...]

$ md5sum pacparser-v1.4.5-ubuntu*zip
a29e84cf7ffa6c2089273215290a6df0  pacparser-v1.4.5-ubuntu-x86_64.zip
eaea12dc1b40958918b8539af4f25260  pacparser-v1.4.5-ubuntu.zip
```

**Reply by manugarg** [reply]
@ckujau No, they were not supposed to be :)

Fixed now. Thanks for checking. I'll add a distribution creating step to the github workflow to make it more standard and error proof.

**Reply by manugarg** [reply]
> @ckujau No, they were not supposed to be :)
> 
> Fixed now. Thanks for checking. I'll add a distribution creating step to the github workflow to make it more standard and error proof.

Changed the workflow to create release binaries that we can upload without any changes:
https://github.com/manugarg/pacparser/pull/203

Also building binaries for MacOS arm64 now.

**Comment by ckujau** [comment]
This works. Thanks again for fixing this so quickly! And, if your GH status is still current: get well soon!
